cvat/apps/analytics_report/permissions.py

```diff
@@ -0,0 +1,80 @@
+from django.conf import settings
```

Comment:
copyright is missing. Need to add.

Reply:
Thanks, updated. I also updated the copyright banners in `*/apps.py`.

Note that since I just moved code from one place to another, I copied the original copyright years from `iam/permissions.py`.